src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4447,6 +4447,8 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
  /fullpaths                    Compiler generates fully qualified paths
  /filealign:&lt;n&gt;                Specify the alignment used for output file 
                                sections
+ /pathmap:&lt;K1&gt;=&lt;V1&gt;,&lt;K2&gt;=&lt;V2&gt;,...      Specify a mapping for source path names output by
```

Comment:
Consider wrapping this at the same column as other lines. I think we are trying to not go beyond 80 columns per line.